anesthetic/samples.py

```diff
@@ -1164,6 +1168,40 @@ def live_points(self, logL=None):
         i = ((self.logL >= logL) & (self.logL_birth < logL)).to_numpy()
         return Samples(self[i]).set_weights(None)
 
+    def n_live(self, i):
+        """
+        Get live points at iteration i.
+
+        Parameters
+        ----------
+        i: i
+            nested sampling iteration
+
+        Returns
+        -------
+        live points at teration i
+
+        """
+        return self.nlive.iloc[i]
+
+    def LX(self, beta, logX):
+        """
+        Get LX, e.g., for Higson plot.
+
+        Parameters
+        ----------
+        beta: float
+            temperature
+        logX: np.ndarray
+            prior volumes
+
+        Returns
+        -------
+         LX: np.ndarray
+        """
+        LX = self.logL*beta + logX
+        return LX
```

Comment:
I don't like the naming of this function, it's misleading. Makes me think that it returns `L * X`, which is not the case.